tests/run_tests.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+function show_help_and_exit()
+{
+    echo "Usage ${SCRIPT} [options]"
+    echo "    options with (*) must be provided"
+    echo "    -h -?          : get this help"
+    echo "    -c             : specify test cases to execute (default: none, executed all matched)"
+    echo "    -d             : specify DUT name (*)"
+    echo "    -e             : specify extra parameter(s) (default: none)"
+    echo "    -f             : specify testbed file (default testbed.csv)"
+    echo "    -i             : specify inventory name"
+    echo "    -k             : specify file log level: error|warning|info|debug (default debug)"
+    echo "    -l             : specify cli log level: error|warning|info|debug (default warning)"
+    echo "    -m             : specify test method group|individual|dump (default group)"
```

Comment:
I think `debug` or `dry_run` might be more descriptive than `dump`. Thoughts?

Reply:
changed to debug. it is dumping test settings anyways. so dry_run is not entirely fit.